docs/platforms/javascript/common/performance/index.mdx

```diff
@@ -17,13 +17,22 @@ If you’re adopting Performance in a high-throughput environment, we recommend
 
 </PlatformCategorySection>
 
+<PlatformContent supported={['javascript']}>
```

Comment:
For non-base browser JS, this really should only say "Tracing is enabled by default", which has no value. We used to show something like "If you use the loader, ..." here for all SDKs, which was not really correct.